msm-benches/Cargo.toml

```diff
@@ -52,6 +52,24 @@ parallel = [
     "rust-kzg-zkcrypto/parallel",
     "rust-kzg-mcl/parallel",
 ]
+strauss = [
+    "rust-kzg-blst/bgmw",
+    "rust-kzg-arkworks4/bgmw",
+    "rust-kzg-arkworks5/bgmw",
+    "rust-kzg-constantine/bgmw",
+    "rust-kzg-zkcrypto/bgmw",
+]
+
+# Re-export diskcache for backends so benches can enable it with a single flag
+# Enables the `diskcache` feature in `kzg` and the common backend crates
+diskcache = [
+    "kzg/diskcache",
```

Comment:
no need to enable directly on kzg, it will be enabled automatically when you enable `diskcache` from any backend